README.md

```diff
@@ -36,7 +36,14 @@ TODO: npm での使用方法を書く
 
 NES.css only provides components. You will need to define your own layout.
 
-The default font is Press Start 2P which only supports English characters. When you're using this framework with any language except English, please use another font.
+The default font is [Press Start 2P](https://fonts.google.com/specimen/Press+Start+2P?selection.family=Press+Start+2P) which only supports English characters. When you're using this framework with any language except English, please use another font. Please follow the [instructions](https://developers.google.com/fonts/docs/getting_started) from Google Fonts about how to include them, or simply include it as below:
+
+```html
+<head>
+    <link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet">
+    <link href="https://unpkg.com/nes.css/css/nes.css" rel="stylesheet" />
+</head>
+```
```

Comment:
Please move sample html to  below Japanese section. 🙏 
because I aslo want to use `as below` (`以下のように`) in Japanese.

~~~diff
NES.cssはコンポーネントのスタイルのみを提供しています。レイアウトはみなさんが好きなように定義してください。

- デフォルトではPress Start 2Pフォントを使っています。英語以外（日本語など）で利用される場合は、別途フォントを読み込んで使ってください。
+ デフォルトのフォントは[Press Start 2P](https://fonts.google.com/specimen/Press+Start+2P?selection.family=Press+Start+2P)を使っています。英語以外（日本語など）で利用される場合は、別途フォントを読み込んで使ってください。 フォントを読み込む方法は、Google Fontsの[Get Started](https://developers.google.com/fonts/docs/getting_started)を参照するか、または以下のように読み込んでください。

```html
<head>
    <link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet">
    <link href="https://unpkg.com/nes.css/css/nes.css" rel="stylesheet" />
</head>
```
~~~

After #46 (split README file) is merged, I will fix this section later. 🔧

Reply:
Adressed in [affa1e89](https://github.com/BcRikko/NES.css/pull/48/commits/affa1e89e2db05feb99e0d9961fd603f5b1ce656) 😄

Reply:
This suggestion includes Japanese section. 👀 

```diff
- デフォルトではPress Start 2Pフォントを使っています。英語以外（日本語など）で利用される場合は、別途フォントを読み込んで使ってください。
+ デフォルトのフォントは[Press Start 2P](https://fonts.google.com/specimen/Press+Start+2P?selection.family=Press+Start+2P)を使っています。英語以外（日本語など）で利用される場合は、別途フォントを読み込んで使ってください。 フォントを読み込む方法は、Google Fontsの[Get Started](https://developers.google.com/fonts/docs/getting_started)を参照するか、または以下のように読み込んでください。
```

Could you change this too? 🙏

Reply:
🙈 [0f90b9b0d]https://github.com/BcRikko/NES.css/pull/48/commits/0f90b9b0dd462d1219f603bb2d1b865ae0113e77)